azurerm/data_source_container_orchestrator_versions.go

```diff
@@ -0,0 +1,92 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+)
+
+func dataSourceArmContainerOrchestratorVersions() *schema.Resource {
+	return &schema.Resource{
+		Read: dataArmSourceContainerOrchestratorVersionsRead,
+
+		Schema: map[string]*schema.Schema{
+			"location": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"orchestrator_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "Kubernetes",
+			},
+			"version_prefix": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"versions": {
+				Type:     schema.TypeList,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+				Optional: true,
```

Comment:
I don't see a `.Get` fo this, so I don't think its `Optional`?

Reply:
These are supposed to be output attributes.
Because of that, I assumed they need to be marked as `Optional` as well. I'll update.

Reply:
Updated.